lib/data/types.json

```diff
@@ -957,7 +597,7 @@
         "vegetation": "BS: SLi, Es, SAh, BAh, MBe ; SS: Hasel, Roter Hornstrauch, Wolliger Schneeball, Berberitze, Schwarzdorn, Weissdorn-Arten, Strauchwicke, Feld-Rose ; KS: Schmerwurz, Maiglöckchen, Bingelkraut, Turm-Gänsekresse, Immenblatt, Breitblättriges Laserkraut, Waldmeister, Gemeine Weisswurz, Schwalbenwurz",
         "transitions": [
           "13e",
-          "16a",
+          "16",
           "22",
           "38.AmÜbergangzuwaldfreienSchutthängenfindetsichofteinHaselgebüsch-SaummitderselbenKrautschicht."
```

Comment:
@ElkeErhardt The new Excel contains this string inside this field. I reckon this is a typo right?

Reply:
@danji90 yes, thats a problem please remove text and i discuss with customer.